General/OOD/SemanticConstructorOverloading/SemanticConstructorOverloading.Problem.Tests/Enum Based Constructor Parameters/Good/SampleClass.Tests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Xunit;
+using FluentAssertions;
+using SemanticConstructorOverloading.Problem.Enum_Based_Constructor_Parameters.Good;
+
+
+namespace SemanticConstructorOverloading.Problem.Tests
+{
+    public class GoodEnumBasedConstructorParametersTests
+    {
+        private readonly ReferenceClass defaultValues = new ReferenceClass("Lukewarm","Computer");
```

Comment:
Instantiate one per every case. Like this you reduce the scope of the object and put it just where it belongs.